### DISCUSSION TITLE
Which blocker do you have in school ?

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1117] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
for me it's Securly

**Comment by nathblade16** [comment]
contentkeeper, specificly contentkeeper express, at least I would if I actually used a school chromebook, typing this on my arch laptop that was once a chromebook but now is an arch laptop

**Reply by C0D3Hermit** [reply]
I also have Lightspeed Classroom.

**Reply by nark0zz-dev** [reply]
NCREN PAC PROXY + DNS + POLICY + DYKNOW + GOPHER BUDDY

**Reply by ghost** [reply]
Goguadian , and iboss

**Comment by FunnyLoaf** [comment]
I have cisco umbrella.

**Reply by Mazedotexe** [reply]
same

**Comment by simpansoftware** [comment]
Lanschool Air, but bypassable with OmadaDNS

**Comment by leoisbadatmakeingthings** [comment]
Blocksi Enterprise Edition. Usually bypass it with skiovox.

**Reply by Ltbeef** [reply]
my Chromebook too updated for that lol, blocksi as well tho

**Comment by Go0gl** [comment]
Zscaler and dyknow

**Reply by nark0zz-dev** [reply]
CMS School?

**Reply by nark0zz-dev** [reply]
Actually, ZScaler is just pac filter. NCREN PAC PROXY is executor for this filter.

**Comment by Solar-Geaux** [comment]
Linewize/Smoothwall

**Reply by Anty847** [reply]
me too

**Reply by alecen43** [reply]
me too :(

**Comment by SniperWasTaken** [comment]
Securly

Just simply locks down everything

**Comment by MyPizzaHasPepperoni** [comment]
Deledao

Uses AI to block sites in real-time

**Reply by thehotgosspotato-debug** [reply]
i used blocksi 2 layer

**Comment by Go0gl** [comment]
same bro

**Comment by luh-99** [comment]
I have mosyle and lightspeed agent. Is there a way to see what things I have on MacOS? I’ve been trying to find macOS exploits like this repository but everything I have came across is for Chromebook. I have a MacBook Air M1 (2020) on Sonoma 14.5 if you need it for something. Also when I try to open terminal it opens then closes really fast, and by fast I mean you can’t even see the menu pop up

**Comment by ghost** [comment]
I have contentkeeper which is really annoying since there's an extension that checks your account and a built-in blocker so ext-hanger only blocks the checker

**Reply by nathblade16** [reply]
ck-e-tp connects you to the proxy and can be defeated using ck-p0isi0n

**Reply by ghost** [reply]
oh neat where can I find that

**Reply by nathblade16** [reply]
> oh neat where can I find that

[contentgiver.github.io](contentgiver)

**Reply by schoolexploitkid** [reply]
https://contentgiver.github.io/guides/ck-p0ision

**Comment by cool-dacool** [comment]
used to be goguardian, they switched to linewize, possibly because of me
